src/main/java/com/Proiect_Varianta_Buna/demo/Controller/CustomerController.java

```diff
@@ -0,0 +1,2 @@
+package com.Proiect_Varianta_Buna.demo.Controller;public class CustomerController {
```

Comment:
Please take a look at Matei PR (https://github.com/MateiPop12/en-summer-apprentice-java/pull/1) to see an example of a valid Controller -> class OrderController -> findAll (metod/endpoint)  / this is a valid example